api/config/dbConnection.js

```diff
@@ -1,20 +1,28 @@
 import mongoose from "mongoose";
-import dotenv from "dotenv";
```

Comment:
I don't get why remove dotenv... Dotenv is for personal use, while the sample-env describes what you have in your dotenv for others to imitate. I don't get why it is removed or what the replacement for dotenv is. Please clarify

Reply:
Dotenv wasn't removed, it will be required/initialised in the main server file. The idea is to import all environment variables into a config object, which will be imported as when needed instead of reaching directly into process.env in multiple places.
The contents of sample-env would be saved in .env in individual repos as it contains the database uri we will be using.